### PR TITLE
Clarify requirement of site connection in README

### PR DESCRIPTION
**Pull request by pestevez**
This PR adds some clarification to the README file about the requirement of having your domain connected for your articles to show up under Publishing Tools.

### CODE REVIEW

**Comment by nittypr007** [comment]
Hi can you help me with instant articles get approved

File: README.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ More likely than not, this is because there is markup in the body of your post t
 Your posts are imported to your library when they are shared on Facebook for the first time.
  
 Alternatively, you can trigger a manual scrape by pasting your URL on our [Share Debugger](http://developers.facebook.com/tools/debug)
+
+Only Instant Articles with URLs in domains you have claimed will show up in the Publishing Tools section.
```
Comment (timjacobi) [inline README.md]: Let's add a link to the docs.

File: README.md [hunk 2]
```diff
@@ -61,7 +63,7 @@ You can propose popular transformer rules to be included in the plugin by [sugge
 
 **How do I post articles to Instant Articles after plugin is installed?**
 
-In order to import your posts to your Instant Articles library on Facebook you need to either share them on your page or use the [Sharing Debugger](https://developers.intern.facebook.com/tools/debug/sharing/) to scrape them. After you have 10 articles imported, you will be able to submit them for review.
+In order to import your posts to your Instant Articles library on Facebook first you need to [Connect Your Site](https://developers.facebook.com/docs/instant-articles/guides/publishertools). Then either share your posts on your page or use the [Sharing Debugger](https://developers.intern.facebook.com/tools/debug/sharing/) to scrape them. After you have 10 articles imported, you will be able to submit them for review.
```
Comment (timjacobi) [inline README.md]: ~~first~~ you need to Connect Your Site **first**

File: README.md (hunk 1 above)
Comment (pestevez) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (pestevez) [reply README.md]: Done, thanks!

**Comment by pestevez** [comment]
Thanks, @timjacobi!

**Pull request merged**